.github/workflows/source.yml

```
@@ -31,3 +31,25 @@ jobs:
       run: |
         sudo apt-get install -y --no-install-recommends doxygen
         .github/workflows/source/doxygen
+  urls:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: urls-checker
+      uses: urlstechie/urlchecker-action@master
```

Can I try action branches like this?
```suggestion
      uses: urlstechie/urlchecker-action@add/certifi
```